dao/src/main/java/org/thingsboard/server/dao/entity/AbstractEntityService.java

```diff
@@ -124,4 +135,15 @@ protected void checkAssignedEntityViewsToEdge(TenantId tenantId, EntityId entity
             }
         }
     }
+
+    protected void setDebugAllUntil(TenantId tenantId, HasDebugMode entity, long now) {
+        int debugDuration = tbTenantProfileCache.get(tenantId).getDefaultProfileConfiguration().getMaxDebugModeDurationMinutes(maxDebugModeDurationMinutes);
```

Comment:
This code does not handle a case of missing tenant.

Reply:
I think its too mach to check "missing tenant" during entity save.